template/src/Autovis/Controls.js

```diff
@@ -0,0 +1,57 @@
+import React from "react";
+import "./controls.css";
+
+const convertSecToTime = secs => {
+  var h = Math.floor(secs / 3600);
+  var m = Math.floor((secs % 3600) / 60);
+  var s = Math.floor((secs % 3600) % 60);
+  return `${h < 10 ? "0" : ""}${h}:${m < 10 ? "0" : ""}${m}:${
```

Comment:
Let's use the pad.ts util for this:
https://github.com/TheActualWalko/autovis-template/blob/master/template/src/Autovis/pad.ts

Reply:
ok done